opus/application/static_media/js/browse.js

```diff
@@ -320,8 +320,8 @@ var o_browse = {
                 slug = "-"+slug;
             }
             let slugIndex = $.inArray(slug, opus.prefs.order);
+            // The clicked-on slug should always be in the order list; this is just a safety precaution
```

Comment:
Sorry, perhaps we weren't entirely clear. The position of the comment needs to move like you did, but the word "this" is confusing. (Yes I know I wrote the comment originally) Let's say "the if is a safety precaution and the condition should always be true".

Reply:
Comments are updated in the latest commit.